packages/maskbook/src/plugins/RedPacket/UI/RedPacketInList.tsx

```diff
@@ -75,3 +141,133 @@ export function RedPacketInList(props: RedPacketInListProps) {
+                                {payload.message === '' ? t('plugin_red_packet_best_wishes') : payload.message}
+                            </Typography>
+                            <Typography variant="body1" className={classNames(classes.info, classes.message)}>
+                                {t('plugin_red_packet_history_duration', {
+                                    startTime: dateTimeFormat(new Date(payload.creation_time * 1000)),
+                                    endTime: dateTimeFormat(
+                                        new Date((payload.creation_time + payload.duration) * 1000),
+                                        false,
+                                    ),
+                                })}
+                            </Typography>
+                            <Typography variant="body1" className={classNames(classes.info, classes.message)}>
+                                {t('plugin_red_packet_history_total_amount', {
+                                    amount: formatBalance(new BigNumber(payload.total), payload.token.decimals, 6),
+                                    symbol: payload.token.symbol,
+                                })}
+                            </Typography>
+                            <Typography variant="body1" className={classNames(classes.info, classes.message)}>
+                                {t('plugin_red_packet_history_split_mode', {
+                                    mode: payload.is_random
+                                        ? t('plugin_red_packet_random')
+                                        : t('plugin_red_packet_average'),
+                                })}
+                            </Typography>
+                        </div>
+                        {canRefund || canSend ? (
+                            <ActionButton
+                                onClick={onSendOrRefund}
+                                className={classes.actionButton}
+                                variant="contained"
+                                size="large">
+                                {canSend ? t('plugin_red_packet_history_send') : t('plugin_red_packet_refund')}
+                            </ActionButton>
+                        ) : null}
+                    </section>
+                    <StyledLinearProgress
+                        barcolor="rgba(44, 164, 239)"
+                        backgroundcolor="rgba(44, 164, 239, 0.2)"
+                        variant="determinate"
+                        value={100 * (1 - Number(payload.total_remaining) / Number(payload.total))}
+                    />
+                    <section className={classes.footer}>
+                        <Typography
+                            variant="body1"
+                            className={classNames(classes.footerInfo, classes.message)}
+                            dangerouslySetInnerHTML={{
```

Comment:
Why

Reply:
I use `<strong>` tag in i18n, `"plugin_red_packet_history_claimed": "Claimed: <strong>{{claimedShares}}/{{shares}}</strong> Share",`, so if do not use dangerouslySetInnerHTML, `<strong>` would be rendered as normal text.

<img src="https://user-images.githubusercontent.com/63733714/108018844-cdaa0400-7053-11eb-8149-6f6e67a5baca.png" width=500 />

Reply:
https://react.i18next.com/latest/trans-component